src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    private final Injector injector = Injector.getInstance("mate.jdbc");
+    private final DriverDao driverDao;
+
+    {
```

Comment:
let's try to use @Inject annotation  before DriverDao driverdao line instead of this block